scripts/build-examples.sh

```diff
@@ -2,8 +2,7 @@
 
 status=""
 
-yarn boostrap &&
-cd examples &&
+yarn bootstrap && cd examples
```

Comment:
Still should have `&&` after the cd to prevent the script continuing if `yarn bootstrap` (or `cd` itself) failed

Reply:
Is that actually necessary? I think that will result in a failure, as is. I haven't seen many scripts use the `&&` so heavily!

Reply:
Ha, it apparently is. TIL.

```sh
asdfasdfasdfasdfasdf
  
echo "hello"
```

fails and then prints `hello`. I'll address